ci/bamboo/build-linux-debug.sh

```diff
@@ -5,11 +5,11 @@ set -o xtrace
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 # Run the common setup
-${DIR}/setup-dependencies.sh
+${DIR}/setup-dependencies-linux.sh
 apt-get install -y iwyu
 
 # Install nupic.core dependencies
-pip install \
+python -m pip install \
```

Comment:
Is invoking pip via the `pip` command as opposed to invoking it as a module a strict requirement?